packages/user_packages/iotcc_mqtt.py

```diff
@@ -31,9 +31,36 @@
 # ----------------------------------------------------------------------------#
 
 from liota.core.package_manager import LiotaPackage
+import time
+import logging
+import random
+
+log = logging.getLogger(__name__)
 
 dependencies = ["general_edge_system", "credentials"]
 
+# --------------------User Configurable Retry and Delay Settings------------------------------#
+
+# The value mentioned below is in multiple of 1K, 2 for 2K, 3 for 3K, 4 for 4K and  5 for 5K Edge Systems
```

Comment:
Add that this should be set based on the total number of gateways being deployed in your infrastructure.Minimum value should be 1. Should we also have a check for that?

Reply:
Done